GNUmakefile

```diff
@@ -66,6 +66,18 @@ EXTERNAL_TOOLS=\
 # list only our .go files i.e. exlcudes any .go files from the vendor directory
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
 
+# docker hub username
+HUB_USER?=openebs
+
+# Repository name
```

Comment:
Should we name these are Image names?
I guess we have named this to repo based on the container image registry. e.g. DockerHub will have these images are repos.

Reply:
yes, So final name will be `$HUB_USER/$*_REPO_NAME`.

Reply:
ok.. can we put these as comments to let new contributors understand this

Reply:
Done!